Libraries/Picker/PickerIOS.ios.js

```diff
@@ -44,21 +46,49 @@ var PickerIOS = React.createClass({
   _stateFromProps: function(props) {
     var selectedIndex = 0;
     var items = [];
+    var styleFromProps = this._getStyleFromProps(props);
+
+    var itemsStyle = {
+      fontSize: styleFromProps.fontSize || 20,
+      color: styleFromProps.color || '#000000',
+      textAlign: styleFromProps.textAlign || 'center'
+    };
+
     ReactChildren.forEach(props.children, function (child, index) {
       if (child.props.value === props.selectedValue) {
         selectedIndex = index;
       }
-      items.push({value: child.props.value, label: child.props.label});
+      items.push({
+        value: child.props.value, 
+        label: child.props.label, 
+        style: itemsStyle
```

Comment:
If the itemStyle applied to each item is always the same, why not pass it once as a single property of the RCTPicker view instead of duplicating the same data for each item?